elfi/methods/posteriors.py

```diff
@@ -0,0 +1,288 @@
+import logging
```

Comment:
I'd prefer that as long as there is just one class, this would be kept in the result module, but you can decide.